tests/behat/features/content_type.civictheme_page.fields.feature

```diff
@@ -68,6 +68,9 @@ Feature: Fields on Page content type
     And I should see text matching "Show last updated date"
     And should see an "input[name='field_c_n_show_last_updated[value]']" element
 
+    And I should see text matching "Custom last updated date"
+    And should see an "input[name='field_c_n_custom_last_updated[0][value][date]']" element
+
```

Comment:
Please add a test to show custom date if the checkbox is checked (in another feature file)